packages/block-library/src/pullquote/style.scss

```diff
@@ -1,8 +1,16 @@
 .wp-block-pullquote {
 	margin: 0 0 1em 0;
 	padding: 3em 0;
-	text-align: center;
+	text-align: center; // Default text-alignment is left.
```

Comment:
Shouldn't that comment read

```suggestion
	text-align: center; // Default text-alignment is center.
```

?

Reply:
Ohh, I think I misread that. The comment expresses what the default alignment would be _without_ this setting, right?

Reply:
This comment is confusing, you're right. I think it was a personal note that I left in there while testing.

I think it should be something like: 

```
// Default text-alignment where the `textAlign` attribute isn't specified.
```